### PR TITLE
[Doc] How to train fast_rcnn model.

### PR DESCRIPTION
**Pull request by zytx121**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Update  fast_rcnn README.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Review by Czm369 — APPROVED** [review]
LGTM

**Review by hhaAndroid — APPROVED** [review]

File: configs/fast_rcnn/README.md [hunk 1]
```diff
@@ -12,6 +12,20 @@ This paper proposes a Fast Region-based Convolutional Network method (Fast R-CNN
...
+```bash
+./tools/dist_test.sh \
+    configs/rpn_r50_fpn_1x_coco.py \
+    checkpoints/rpn_r50_fpn_1x_coco_20200218-5525fa2e.pth \
+    8 \
+    --out rpn_r50_fpn_1x_test2017.pkl
+```
+
+Then, you also need to modify the path of test set in the RPN config in order to generate `rpn_r50_fpn_1x_train2017.pkl` and `rpn_r50_fpn_1x_val2017.pkl`.
+Finally, setting the path of proposals in your Fast R-CNN config. And you can start training the Fast R-CNN now.
```
Comment (ZwwWayne) [inline configs/fast_rcnn/README.md:27]: show the concrete example below.

File: configs/fast_rcnn/README.md [hunk 2]
```diff
@@ -12,6 +12,20 @@ This paper proposes a Fast Region-based Convolutional Network method (Fast R-CNN
...
+Before training the Fast R-CNN, you should first train an [RPN](../rpn/README.md), and use the RPN to extract the region proposals of the test set by this command:
+```bash
+./tools/dist_test.sh \
+    configs/rpn_r50_fpn_1x_coco.py \
+    checkpoints/rpn_r50_fpn_1x_coco_20200218-5525fa2e.pth \
+    8 \
+    --out rpn_r50_fpn_1x_test2017.pkl
+```
+
+Then, you also need to modify the path of test set in the RPN config in order to generate `rpn_r50_fpn_1x_train2017.pkl` and `rpn_r50_fpn_1x_val2017.pkl`.
```
Comment (ZwwWayne) [inline configs/fast_rcnn/README.md:26]: may show some examples.

File: configs/fast_rcnn/README.md [hunk 3]
```diff
@@ -12,6 +12,20 @@ This paper proposes a Fast Region-based Convolutional Network method (Fast R-CNN
 <img src="https://user-images.githubusercontent.com/40661020/143882189-6258c05c-f2a1-4320-9282-7e2f2d502eb2.png"/>
 </div>
 
+## Introduction
+
+Before training the Fast R-CNN, you should first train an [RPN](../rpn/README.md), and use the RPN to extract the region proposals of the test set by this command:
```
Comment (ZwwWayne) [inline configs/fast_rcnn/README.md:17]: You -> Users.

Comment (ZwwWayne) [inline configs/fast_rcnn/README.md:17]: this command -> the command as below:

**Comment by mukaiNO1** [comment]
I got train2017.pkl file, but it also show me a error

**Comment by zytx121** [comment]
Hi @mukaiNO1 
You should add `proposal_file` in the dict of train/val/test, not outside of them.

**Pull request merged**